packages/manager/modules/pci-object-storage/src/object-storage.service.js

```diff
@@ -151,4 +151,36 @@ export default class PciStoragesObjectStorageService {
       },
     );
   }
+
+  getCustomerRegions(serviceName, withPaginationMode = false) {
+    const requestHeader = withPaginationMode
+      ? {
+          'X-Pagination-Mode': 'CachedObjectList-Pages',
```

Comment:
If the number of region is high, you might need to add `x-pagination-number` and `x-pagination-size` parameters as well